src/targets/github.ts

```diff
@@ -422,6 +334,24 @@ export class GithubTarget extends BaseTarget {
     }
   }
 
+  /**
+   * Publishes the release draft.
+   *
+   * @param release_id Release object ID
+   */
+  public async publishReleaseDraft(release_id: number) {
+    if (isDryRun()) {
```

Comment:
Same here, but your call. We already have this style in the `deleteAsset` function above.

Reply:
Done in d7886d943388383dddec2db271e371d29488d4f8.